app/coffeescripts/gradebook2/Turnitin.coffee

```diff
@@ -25,7 +25,7 @@ define [
       data.state = stateList[max(states)]
       data
 
-    extractDataFor: (submission, key, urlPrefix) ->
+    extractDataForTurnitin: (submission, key, urlPrefix) ->
```

Comment:
This should be named more generically and either take a switch or some other augment to switch between "Turnitin" and "Vericite" modes